.gitignore

```diff
@@ -1 +1,2 @@
-**./.darp
\ No newline at end of file
+**./.darp
+keys/rsa-private-key.pem
```

Comment:
xdd smart

Reply:
It is a trap, there is k8s/crypto/key-configmap.yaml now too